index.css

```diff
@@ -0,0 +1,193 @@
+body{
+  font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+  background-attachment: fixed;
+  background-image: url('http://www.fleming.kyschools.us/userfiles/316/Clipart%20Images/math.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin:auto;
+}
+.dock{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  top: 27%;
+  left: 25%;
+}
+/*
+.calculator-title{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+}*/
+.calculator-body{
```

Comment:
please redo the styles for your calculator using static pixel values that exactly match the OX calculator. You should be measuring these buttons on the screen and making sure they match as pixel perfect as you can.  

this: is not close enough
![screen shot 2017-02-09 at 3 41 00 pm](https://cloud.githubusercontent.com/assets/8385/22807885/33f74bf6-eede-11e6-9c56-f8da1a043133.png)